main/webapp/modules/core/langs/translation-en.json

```diff
@@ -749,5 +749,23 @@
     "core-buttons/match": "Match",
     "core-buttons/copy": "Copy",
     "core-buttons/transpose": "Transpose",
-    "core-buttons/apply-to-all": "Apply to All Identical Cells"
+    "core-buttons/apply-to-all": "Apply to All Identical Cells",
+    "core-import-formats/text/line-based": "Line-based text files",
+    "core-import-formats/text/line-based/*sv": "CSV / TSV / separator-based files",
+    "core-import-formats/text/line-based/fixed-width": "Fixed-width field text files",
+    "core-import-formats/text/rdf/nt": "RDF/N-Triples files",
+    "core-import-formats/text/rdf/n3": "RDF/N3 files",
+    "core-import-formats/text/rdf/ttl": "RDF/Turtle files",
+    "core-import-formats/text/rdf/xml": "RDF/XML files",
+    "core-import-formats/text/rdf/ld+json": "JSON-LD files",
+    "core-import-formats/text/xml": "XML files",
+    "core-import-formats/binary/text/xml/xls/xlsx": "Excel files",
+    "core-import-formats/text/xml/ods": "Open Document Format spreadsheets (.ods)",
+    "core-import-formats/text/json": "JSON files",
+    "core-import-formats/text/marc": "MARC files",
+    "core-import-formats/text/wiki": "Wikitext",
+    "core-import-formats/text/line-based/pc-axis": "PC-Axis text files",
```

Comment:
The `text/line-based/pc-axis` format is defined in the `pc-axis` extension, so this translation message should be in that extension, not in the main Butterfly module.

Reply:
Then can I make new folder of name langs in extensions/pc-axis/module/langs

Reply:
its not a good idea

Reply:
Why is it not a good idea?

Reply:
yes we can do that

Reply:
![Screenshot from 2020-08-28 12-56-22](https://user-images.githubusercontent.com/63091769/91533846-34185a00-e92e-11ea-8560-d7a978aa6063.png)

Reply:
Thanks! You will need to load the translations from the extension. You can just do like other extensions do:

https://github.com/OpenRefine/OpenRefine/blob/master/extensions/wikidata/module/scripts/menu-bar-extension.js#L2-L16

https://github.com/OpenRefine/OpenRefine/blob/master/extensions/database/module/scripts/index/database-import-controller.js#L34-L46